apps/client/app/interfaces.tsx

```diff
@@ -0,0 +1,31 @@
+// Moved interfaces here to avoid code replication
+
+export interface Item {
+    id: number;
+    name: string;
+    sku: string;
+}
+
+export interface ParentItem {
+    id: number;
+    name: string;
+    items: Item[];
+}
+
+export interface PurchaseOrder {
+    id: number;
+    vendor_name: string;
+    // Decided to define the dates here as strings to avoid unnecessary back and forth conversion from Date to string since the UI needs it to be a string for display ane editing purposes
```

Comment:
Wanted to highlight this comment in case there are concerns with my approach here.